packages/eslint-plugin-template/docs/rules/prefer-self-closing-tags.md

```diff
@@ -690,7 +690,7 @@ The rule does not have any configuration options.
 #### ✅ Valid Code
 
 ```html
-<my-component>  <!-- not empty -->  </ng-container>
+<my-component>  <!-- not empty -->  </my-component>
```

Comment:
ℹ️ This PR will be blocked until #2256 is merged. The suppressed parse error was allowing this valid test case to pass when the rule actually treats the element as empty.